app/src/main/java/fi/project/petcare/ui/screens/PetProfileFormScreen.kt

```diff
@@ -1,20 +1,252 @@
+            },
+            label = { Text("Species") },
+            leadingIcon = { Icon(imageVector = Icons.Default.Animation, contentDescription = null) },
+            isError = speciesError.isNotEmpty(),
+            singleLine = true,
+            modifier = Modifier.padding(bottom = 8.dp)
+        )
+        Text(
+            text = speciesError,
+            color = Color.Red,
+            fontSize = 12.sp,
+            modifier = Modifier.padding(start = 16.dp)
+        )
+
+        // Breed field with validation
+        OutlinedTextField(
+            value = breed,
+            onValueChange = {
+                breed = it
+                breedError = if (it.isEmpty()) "Breed cannot be empty" else ""
+            },
+            label = { Text("Breed") },
+            leadingIcon = { Icon(imageVector = Icons.Default.Pets, contentDescription = null) },
+            isError = breedError.isNotEmpty(),
+            singleLine = true,
+            modifier = Modifier.padding(bottom = 8.dp)
+        )
+        Text(
+            text = breedError,
+            color = Color.Red,
+            fontSize = 12.sp,
+            modifier = Modifier.padding(start = 16.dp)
+        )
+
+        // Birthdate field with validation
+        OutlinedTextField(
+            value = age_months,
+            onValueChange = {
+                age_months = it
+                age_monthsError = if (it.isEmpty()) "Age cannot be empty" else ""
+            },
+            label = { Text("Age") },
+            leadingIcon = { Icon(imageVector = Icons.Default.CalendarViewMonth, contentDescription = null) },
+            isError = age_monthsError.isNotEmpty(),
+            singleLine = true,
+            modifier = Modifier.padding(bottom = 8.dp)
+        )
+        Text(
+            text = age_monthsError,
+            color = Color.Red,
+            fontSize = 12.sp,
+            modifier = Modifier.padding(start = 16.dp)
+        )
+
+        Spacer(modifier = Modifier.height(16.dp))
+
+        // Save button
+        Button(
+            onClick = { /* Save the form data if validation passes */ },
+            modifier = Modifier
+                .fillMaxWidth()
+                .heightIn(48.dp),
+            contentPadding = PaddingValues(),
+            colors = ButtonDefaults.buttonColors(Color.Transparent)
+        ) {
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .heightIn(48.dp)
+                    .background(
+                        brush = Brush.horizontalGradient(
+                            listOf(
+                                md_theme_dark_onPrimary,
+                                md_theme_light_primary
+                            )
+                        ),
+                        shape = RoundedCornerShape(50.dp)
+                    ),
+                contentAlignment = Alignment.Center
+            ) {
```

Comment:
This is a nice-looking button with a gradient color as the background. However, wouldn't the plain color offered by default be a better fit? It would align more cohesively with the rest of the styles used throughout the app.